### ISSUE TITLE
Mac travis builds are broken

### ISSUE THREAD

**Issue by peastman** [pandegroup/openmm#1714]
For a little while now, the Mac travis builds have been failing due to a segfault in doxygen.  Any idea what might be causing it?

**Comment by jchodera** [comment]
[Here's the segfault](https://travis-ci.org/pandegroup/openmm/jobs/190804454#L2165-L2191), for reference:
```
Generating XML output for class OpenMM::Context
/bin/sh: line 1:  5342 Segmentation fault: 11  /usr/local/bin/doxygen
make[2]: *** [wrappers/doxygen/xml/index.xml] Error 139
make[2]: *** Deleting file `wrappers/doxygen/xml/index.xml'
make[1]: *** [wrappers/CMakeFiles/ApiWrappers.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[  0%] Built target AmoebaApiWrappers
make: *** [all] Error 2
```

**Comment by jchodera** [comment]
The first thing that comes to mind is that maybe there's an issue with the current `brew` version of `doxygen` with the [default `xcode7.3` travis `osx` image](https://docs.travis-ci.com/user/osx-ci-environment/#OS-X-Version). I'll try opening a PR with a more recent `xcode` version.

**Comment by jchodera** [comment]
Switching to `xcode8.2` didn't get rid of the segfault.

Hm, it looks like the homebrew `doxygen` bottle recipe was changed on [5 Jan 2017](https://github.com/Homebrew/homebrew-core/commit/e517eaca549280811c36ad9250ae9e4e1f71aa43). Does that roughly correspond to when things started failing?

**Comment by peastman** [comment]
The first time I specifically noted this problem was in #1708, which was on Jan. 4.  Perhaps the two dates are for different time zones?

**Comment by jchodera** [comment]
I'm seeing a bunch of warnings like these when doxygen 1.8.13 is used. I wonder if these are relevant:
```
[  0%] Parsing OpenMM header files with Doxygen...
[  0%] Parsing OpenMM header files with Doxygen...
warning: Tag `SYMBOL_CACHE_SIZE' at line 289 of file `Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag `SYMBOL_CACHE_SIZE' at line 289 of file `Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag `SHOW_DIRECTORIES' at line 473 of file `Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag `SHOW_DIRECTORIES' at line 473 of file `Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag `HTML_ALIGN_MEMBERS' at line 803 of file `Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag `HTML_ALIGN_MEMBERS' at line 809 of file `Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag `XML_SCHEMA' at line 1144 of file `Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag `XML_DTD' at line 1150 of file `Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag `XML_SCHEMA' at line 1150 of file `Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag `XML_DTD' at line 1156 of file `Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: doxygen no longer ships with the FreeSans font.
You may want to clear or change DOT_FONTNAME.
Otherwise you run the risk that the wrong font is being used for dot generated graphs.
warning: doxygen no longer ships with the FreeSans font.
You may want to clear or change DOT_FONTNAME.
Otherwise you run the risk that the wrong font is being used for dot generated graphs.
```

**Comment by peastman** [comment]
Those are just warnings about obsolete options in the configuration file.  They wouldn't explain a segfault.

**Comment by peastman** [comment]
Presumably we can use the same fix as https://github.com/omnia-md/conda-recipes/pull/687.

**Comment by jchodera** [comment]
> Presumably we can use the same fix as omnia-md/conda-recipes#687.

Yes, go for it!
